threescale/api.go

```diff
@@ -123,7 +123,8 @@ func (c *Client) doReportReq(req *http.Request, values url.Values) (*ReportRespo
 	}
 	defer resp.Body.Close()
 
-	if resp.StatusCode != http.StatusAccepted {
+	// ensure response is in 2xx range
+	if !(resp.StatusCode >= http.StatusOK && resp.StatusCode < http.StatusMultipleChoices) {
```

Comment:
I'd expect there would be some helper to do this rather than effectively knowing the relation between these constants and their relative numbers and semantics. This is really not much better if at all than encoding the numbers directly. It is even probably worse from a semantics POV.